2022q4/wifi.adoc

```diff
@@ -0,0 +1,34 @@
+=== Wireless updates
+
+Links: +
+link:https://people.freebsd.org/~bz/wireless/[Bjoern's Wireless Work In Progress landing page] URL: link:https://people.freebsd.org/\~bz/wireless/[https://people.freebsd.org/~bz/wireless/] +
+
+Contact: Bjoern A. Zeeb <bz@FreeBSD.org>
+
+During the quarter not much work was publicly visible and admittedly slightly slow.
+Behind the scenes wireless work was happening on two fronts:
+
+ * 11n, 11ac, and wpa,
+
+ * more drivers and firmware, problems, testing, and filling gaps for these.
+
+While the main development for newer standards and the Intel iwlwifi driver is sponsored by the FreeBSD Foundation, I find myself spending a lot of private time with Realtek rtw88 and rtw89, Atheros, and Mediatek mt76 (7921 and 7915) drivers now as well.
+Testing changes on bare metal and using bhyve VMs using passthru has become more time-consuming, with the amount of supported chipsets increasing, in order not to break other drivers.
+Even different (generations of) chipsets supported by the same driver, at times, behave differently with the same change.
+
+A separate discussion was brought to me about the size of firmware added to the tree for the already existing drivers and the firmware for more drivers coming.
+The chicken-egg problem to solve is having firmware available on the release media; without firmware, a lot of modern laptops will not have any sort of outside communications available at the time of install in their default configuration.
```

Comment:
@bzfbd please, do you mean that the problem began (in the second quarter) when `13.1-RELEASE` was released; that (without the `RELEASE` medium context) there's no problem with the size of firmware added to the tree?

I imagine not; this is part of what's not clear.